src/ya-comiste-react-native/src/relay/relayEnvironment.js

```diff
@@ -1,15 +1,30 @@
 // @flow
 
 import { Environment, Network, RecordSource, Store } from 'relay-runtime';
+import fetch from '@adeira/fetch';
 
-function fetchQuery(operation, variables) {
-  // TODO: specify JSON.schema
-  if (variables.entrypointID === 'com.yaComiste.Explore') {
-    return require('./mocks/com.yaComiste.Explore.json');
-  } else if (variables.entrypointID === 'com.yaComiste.ExploreDetail') {
-    return require('./mocks/com.yaComiste.ExploreDetail.json');
+export const handleData = (response: Response): Promise<$FlowFixMe> | Promise<string> => {
+  const contentType = response.headers.get('content-type');
+  if (contentType != null && contentType.indexOf('application/json') !== -1) {
```

Comment:
doesn't `headers.get` return string or null? Why not just check contentType === 'application/json'?

Reply:
This function was unused (copy-pasted by accident) so I completely removed it. 🙂